src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,10 @@
+import { TodoInfo } from '../TodoInfo';
+
 // Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+export const TodoList = ({ todos }) => (
+  <>
+    <section className="TodoList">
+      {todos.map(todo => (<TodoInfo todo={todo} />))}
```

Comment:
separate line please 
And you forgot a key
```suggestion
      {todos.map(todo => (
        <TodoInfo todo={todo} key={todo.id} />
      ))}
```